easybuild/easyblocks/generic/intelbase.py

```diff
@@ -164,7 +164,7 @@ def clean_home_subdir(self):
             for tree in os.listdir(self.home_subdir_local):
                 self.log.debug("... removing %s subtree" % tree)
                 path = os.path.join(self.home_subdir_local, tree)
-                if os.path.isfile(path):
+                if os.path.isfile(path) or os.path.islink(path):
                     os.remove(path)
```

Comment:
use `remove_file` instead from `filetools` (we don't have a `remove_dir` yet, and if we do, we should also add a `remove` that wraps around both `remove_file` and `remove_dir`)